File: 3360-stone-removal-game/3360-stone-removal-game.java
class Solution {
    public boolean canAliceWin(int n) {
        int stone = 10;
        while (n >= 0) {
            if (n >= stone) {
                n -= stone;
                stone--;
            } else {
                return false;
            }
            if (n >= stone) {
                n -= stone;
                stone--;
            } else {
                return true;
            }
        }
        return true;
    }
}

File: LeetCode/3360-stone-removal-game/3360-stone-removal-game.py
class Solution:
    def canAliceWin(self, n: int) -> bool:
        stone = 10
        while n >= 0:
            if n >= stone:
                n -= stone
                stone -= 1
            else:
                return False
            if n >= stone:
                n -= stone
                stone -= 1
            else:
                return True
        return True